.ci/job/job_runner.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir="$(dirname $0)/../"
+source "$cidir/lib.sh"
+
+#
+# Variables that the user can overwrite
+#
+KATA_TESTS_JOB_RUNNER="${KATA_TESTS_JOB_RUNNER:-vm}"
+
+#
+# Script variables
+#
+
+ALL_AVAILABLE_JOBS=(
+	"baremetal-pmem"
+	"cri_containerd_k8s"
+	"cri_containerd_k8s_initrd"
+	"cri_containerd_k8s_complete"
+	"cri_containerd_k8s_minimal"
+	"crio_k8s"
+	"crio_k8s_complete"
+	"crio_k8s_minimal"
+	"cloud-hypervisor-k8s-crio"
+	"cloud-hypervisor-k8s-containerd"
+	"cloud-hypervisor-k8s-containerd-minimal"
+	"Ccloud-hypervisor-k8s-containerd-fullL"
+	"firecracker"
+	"vfio"
+	"virtiofs_experimental"
+	"metrics"
```

Comment:
mmm I think that these jobs will be hard to reproduce as they run in baremetal and even if somebody runs the tests locally...the results will not match

Reply:
As long as the host allows for nested virt, I think it should be workable.